### PR TITLE
Only send to active susbcriptions

### PR DESCRIPTION
**Pull request by scruti**
As part of [Trello task](https://trello.com/c/RqlIP4NA/37-no-enviar-emails-a-suscripciones-con-activo-false) I had to filter the news tematica to be sent filtering by active subscriptions.

### CODE REVIEW

**Comment by scruti** [comment]
Massive replacements to update `FactoryGirl` and `FFaker` calls. Ignoring Hound.

File: spec/controllers/news_tematica/news_tematicas_controller_spec.rb [hunk 1]
```diff
@@ -5,28 +5,26 @@
...
   let(:dominio) { 'test.host' }
-  let(:admin) { FactoryGirl.create(:admin) }
+  let(:admin) { create(:admin) }
 
   describe "contenidos_elegidos" do
     let(:mensaje_raso) do
-      mensaje = FactoryGirl.create(:tema, created_at: 3.days.ago, titulo: 'CAF paga dividendo hoy')
-      mensaje.tap { |mensaje| FactoryGirl.create(:veces_leido, leido: mensaje, contador: 100) }
+      atributos = { created_at: 3.days.ago, titulo: 'CAF paga dividendo hoy' }
+      create(:tema, atributos).tap { |mensaje| create(:veces_leido, leido: mensaje, contador: 100) }
```
Comment (Fernan2) [inline spec/controllers/news_tematica/news_tematicas_controller_spec.rb]: :+1:

**Comment by Fernan2** [comment]
+1

**Comment by d2bit** [comment]
:+1: I didn't know about `yielding` on rspec

**Pull request merged**